core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -1235,7 +1236,7 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
 
   private def deleteLog(fs: FileSystem, log: Path): Boolean = {
     var deleted = false
-    if (isBlacklisted(log)) {
+    if (!isAccessible(log)) {
```

Comment:
I thought should function should be isInAccessible or you have to change its logic. You added the ! but the isAccessible function does the same thing the isBlacklisted did.  Probably why tests are failing.

Reply:
Thanks for the catch, my mistake. You're right that I remembered to update the callsites but not the definition.